lib/extractor/oci-archive/index.ts

```diff
@@ -0,0 +1,15 @@
+import { OciArchiveManifest } from "../types";
+
+export { extractArchive } from "./layer";
```

Comment:
[Minor point]
I think I'm missing something. Are we using this extractArchive here?